lib/auth/auth.go

```diff
@@ -91,6 +91,10 @@ const (
 	MaxFailedAttemptsErrMsg = "too many incorrect attempts, please try again later"
 )
 
+const (
+	githubCacheTimeout = time.Hour
```

Comment:
Nit: Add a godoc explaining what this is for.

Reply:
Done